openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -183,10 +184,18 @@ def create_placeholder(self, placeholder_data):
         placeholder_name = self._create_placeholder_name(placeholder_data)
 
         placeholder = cmds.spaceLocator(name=placeholder_name)[0]
-        if parent:
-            placeholder = cmds.parent(placeholder, selection[0])[0]
 
-        imprint(placeholder, placeholder_data)
+        # TODO: this can crash if selection can't be used
+        cmds.parent(placeholder, selection[0])
```

Comment:
This will indeed fail to work without `parent`. But why not just keep the `if statement` logic that was there before. That code seems just fine. 
Except that I'd write it as:

```suggestion
        if parent:
            placeholder = cmds.parent(placeholder, parent)[0]
```